README.md

```diff
@@ -463,6 +463,48 @@ anon = LocalCKAN(username='')
 print(anon.action.status_show())
 ```
 
+#### Extra Loggging
+
+To enable extra debug logging for the execution of LocalCKAN ckanapi commands, you can enable the config option in your CKAN INI file.
+
+```
+ckanapi.log_local = True
+```
+
+The output of the log will look like:
+
+```
+DEBUG [ckanapi.cli.main] OS User <user> executed LocalCKAN: ckanapi <args>
+```
+
+Because the ckanapi namespace does not match ckan or ckanext namespaces, you may need to create a new logging handler. Example:
```

Comment:
we could log as `ckan.ckanapi` to avoid all this extra work, right? (ckanapi with LocalCKAN is technically running as a ckan subcommand so I think it's appropriate)

Reply:
Okay, explicitly set the logging name to `ckan.ckanapi`. And changed the log level from `debug` to `info`